hack/verify-dist.sh

```diff
@@ -5,20 +5,80 @@ set -e
 RED='\033[0;31m'
 NOCOLOR='\033[0m'
 GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Comment:
Are you intending to use the `os::log::*` functions that you pulled in with this PR?

Reply:
yup. updated.